Binance.Net/BinanceClient.cs

```diff
@@ -186,18 +195,24 @@ public async Task<WebCallResult<DateTime>> GetServerTimeAsync(bool resetAutoTime
                 var localTime = DateTime.UtcNow;
                 var result = await ExecuteRequest<BinanceCheckTime>(url).ConfigureAwait(false);
                 if (!result.Success)
+                {
```

Comment:
I don't really like the added brackets for single line statements. Could you revert this?

Reply:
It auto-formatting by visual studio.
I will try to revert it.